examples/tv-app/android/java/AppPlatformShellCommands-JNI.cpp

```diff
@@ -57,7 +57,7 @@ static CHIP_ERROR pairApp(bool printHeader, size_t index)
     if (state == nullptr)
     {
         char str[64];
-        sprintf(str, "udc client[%d] null \r\n", index);
+        sprintf(str, "udc client[%ld] null \r\n", static_cast<long>(index));
```

Comment:
@andy31415 This should probably have just cast to `unsigned long` instead of `long`.....